packages/fast-usdc/test/exos/transaction-feed.test.ts

```diff
@@ -117,15 +117,15 @@ test('disagreement after publishing', async t => {
   });
 });
 
-test('disabled operator', async t => {
+test('remove operator', async t => {
   const feedKit = makeFeedKit();
   const { op1 } = await makeOperators(feedKit);
   const evidence = MockCctpTxEvidences.AGORIC_PLUS_OSMO();
 
   // works before disabling
   op1.operator.submitEvidence(evidence);
 
-  op1.admin.disable();
+  await feedKit.creator.removeOperator('op1');
```

Comment:
maybe this means `makeOperator` can lose the awkward `admin` thing altogether?

Reply:
`removeOperator` calls `operatorKit.admin.disable()`.

What's awkward about it?

Reply:
I suggest that result of `makeOperator` in the test file should not include an `.admin` property; the returned value should represent what an operator can do, and operators don't have `admin` authority.

Reply:
I think the change you're asking for is to app, not just tests.

The current design is that operators do get the admin facet of the operator kit. (My recollection from earlier; I'm on a phone now)

Right now that means they can disable themselves. Not a bad thing but I agree they shouldn't get a facet called admin.

So in this PR I'll attenuate the kit to omit admin in the offer result.